make.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function make(...makeArgs) {
+    return function(...args) {
+        if (typeof args[0] === 'function') {
+            return makeArgs.reduce(args[0], 0);
```

Comment:
Initializing reduce accumulator to 0 here is disputable. From one side, it gives gives you zero in result when the numbers for make are not provided at all, only the function (I see a test for this case in your test suite). But from the other side, this will provide wrong results in some cases, e.g. for function like (a,b) => a * b. The only test in suite with such function already contains 0 as one of the arguments to make, so you haven't noticed this problem.